docs-src/cloud/tcld/namespace/retention.md

```diff
@@ -0,0 +1,40 @@
+---
+id: retention
+title: tcld namespace retention
+sidebar_label: retention
+description: How to manage the length of time a closed Workflow is preserved before deletion.
+tags:
+  - tcld
+---
+
+The `tcld namespace retention` command manages the length of time (in days) a closed Workflow is preserved before deletion for a given Namespace in Temporal Cloud.
+
+Alias: `r`
+
+`tcld namespace retention`
+
+The following modifier controls the behavior of the command.
```

Comment:
`set` and `get` are commands instead of modifiers. Suggest restructuring similar to `tcld namespace search-attributes`, giving `set` and `add` separate Markdown files and specifying the modifiers. (Conveniently, `--namespace` can be a straightforward copy-and-paste.)